### ISSUE TITLE
Client-side SWF cache in split-page mode (feature request)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#239]
```
When scrolling back and forth in a split-page mode document, the client reloads 
pages that it already retrieved. The wait time for this can greatly decrease 
the user experience -- an option for the viewer to cache all previously loaded 
pages would be great to see.
```

Original issue reported on code.google.com by `kmewh...@gmail.com` on 26 Nov 2011 at 7:29

**Comment by GoogleCodeExporter** [comment]
```
Fixed in upcoming version 1.48
```

Original comment by `erik.eng...@devaldi.com` on 28 Nov 2011 at 1:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Could you provide a link to the parameter that should be set to cache split 
pages?  Or the commit in which this was fixed?  We are still seeing previously 
loaded pages get reloaded each time they are viewed with FlexPaper 1.5.0.
```

Original comment by `b...@brandonturner.net` on 31 Jan 2012 at 10:03
